lib/iris/analysis/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2016, Met Office
+# (C) British Crown Copyright 2010 - 2017, Met Office
```

Comment:
Oooh, I see now that #2290 is actually touching ```analysis/__init__.py``` but doesn't update the header, so that fix really belongs there.
I will remove that change from here -- it should still fix the v1.12.x branch.